packages/eslint-config-fb-strict/package.json

```diff
@@ -8,8 +8,9 @@
   "license": "BSD-3-Clause",
   "dependencies": {
     "babel-eslint": "^7.0.0",
-    "eslint": "^3.3.0",
+    "eslint-config-fbjs": "^2.0.0-alpha.1",
     "eslint-plugin-babel": "^4.0.0",
-    "eslint-config-fbjs": "^1.0.0"
+    "eslint-plugin-jest": "^18.1.0",
+    "eslint": "^3.3.0"
```

Comment:
doing `npm install -D something` will sort it alphabetically again, that diff might get annoying